src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -484,8 +484,9 @@ gl_sarray style_transfer::convert_style_indices_to_filter(
 
 gl_sframe style_transfer::predict(variant_type data,
                                   std::map<std::string, flexible_type> opts) {
-  gl_sframe_writer result({"style_idx", "stylized_image"},
```

Comment:
Is this needed? Bad merge?

Reply:
We are changing the output format of the SFrame, so this should be needed?

Reply:
My bad, sorry.